### PR TITLE
Awesome project

### PR DESCRIPTION
**Pull request by kensayo**
**Awesome Books**

- Create a collection that keeps a list of books
- Create a function to add a new book to the collection, with title and author.
- Display all books saved in the collection in the top part of the page.
- Make sure that when a user clicks the "Add" button:
- A new book is added to the collection.
- The new book is displayed in the page.
- The correct book is removed from the collection.
- The correct book disappears from the page.

### CODE REVIEW

File: js/file.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  const element = document.getElementById(index);
+  element.parentNode.removeChild(element);
+
+  window.localStorage.setItem('cataloge', JSON.stringify(books));
+}
+/* eslint-enable */
+
+function showBooks() {
+  for (let i = 0; i < books.length; i += 1) {
```
Comment (carlos-ssh) [inline js/file.js:26]: **Your project isn't working well i can't display books.**
- Kindly fix errors displayed on chrome console related to line 21 of this file.

File: js/file.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+let books = JSON.parse(localStorage.getItem('cataloge'));
+const file = document.querySelector('.book');
+
+function addBooks(ti, aut) {
+  books.push({ title: ti, author: aut, id: books.length });
+  window.localStorage.setItem('cataloge', JSON.stringify(books));
+}
+
+/* eslint-disable */
```
Comment (carlos-ssh) [inline js/file.js]: - Implement a well use of your linter. Try to use the right rule without disable eslint.

File: js/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
```
Comment (carlos-ssh) [inline js/README.md]: **Your README file isn't displayed well in your repository. Kindly fix it.**

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Davidosky007** [comment]
## Hi @kensayo @mike2611 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
CR1 GIF

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**